__tests__/app.test.js

```diff
@@ -118,7 +118,89 @@ describe("GET /api/articles", () => {
         expect(datedArticles).toBeSortedBy("created_at", { descending: true });
       });
   });
-});
+  test("200: The articles can be queried to be sorted by any valid column, default created_at and order descending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("article_id", { descending: true });
+      })
+      .then(() => {
+        return request(app)
+          .get("/api/articles?sort_by=votes")
+          .expect(200)
+          .then(({ body: { articles } }) => {
+            expect(articles).toBeSortedBy("votes", { descending: true });
+          });
+      })
+      .then(() => {
+        return request(app)
+          .get("/api/articles?sort_by=author")
+          .expect(200)
+          .then(({ body: { articles } }) => {
+            expect(articles).toBeSortedBy("author", { descending: true });
+          });
+      });
+  });
```

Comment:
This is an interesting way to test for this, but you dont have to have this robust of a test, because when combined with your error handling tests we know anything outside of what we expect will throw an error and this is then indirectly tested